### ISSUE TITLE
travis CI fails to trigger heroku rebuild

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/nextstrain.org#7]
Travis CI is supposed to trigger `npm run redeploy-site` -- [travis yaml file here](https://github.com/nextstrain/nextstrain.org/blob/master/.travis.yml) -- on successful pushes to master. However it fails to do so. Manually running `npm run redeploy-site` is successful.

Travis CI log error:
```
$ if [[ "$TRAVIS_PULL_REQUEST" == "false" && "$TRAVIS_BRANCH" == "master" ]]; then npm run redeploy-site fi
/home/travis/.travis/job_stages: eval: line 105: syntax error: unexpected end of file
Done. Your build exited with 0.
```

Note that auspice (prior to v1.35) was able to trigger the nextstrain.org repo to rebuild successfully using a [similarly formatted yaml file](https://github.com/nextstrain/auspice/blob/v1.34.2/.travis.yml). Its successful travis CI log:
```
$ if [[ "$TRAVIS_PULL_REQUEST" == "false" && "$TRAVIS_BRANCH" == "release" ]]; then npm run gzip-and-upload; npm run redeploy-site; npm run rebuild-docker-image; npm config set //registry.npmjs.org/:_authToken="$NPM_TOKEN"; npm publish; fi
> auspice@1.34.3 gzip-and-upload /home/travis/build/nextstrain/auspice
> ./scripts/gzip-and-upload.sh
...
> ./scripts/rebuild-docker-image.sh
Pinging Travis CI to rebuild Docker image
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   656  100   564  100    92   2067    337 --:--:-- --:--:-- --:--:--  2073
...
```

**Comment by jameshadfield** [comment]
Note that automatic heroku deployment may not be desirable anymore (I don't think it ever worked from this repo).

**Comment by trvrb** [comment]
I can take a look at this. Thanks for adding the issue.

**Comment by trvrb** [comment]
This has been resolved via a combination of `0709ca12ab8af2f2f73006e8b904bce953a2e971` (missing comma) and replacing the Heroku auth token environmental variable in Travis.
